### ISSUE TITLE
Error loading esdoc config after updating

### ISSUE THREAD

**Issue by mmoss** [esdoc/esdoc#74]
It looks like commit c796ac2fd7e1816a08f4677411b6a050897ed96b has broken the loading of our esdoc config:

```
.esdoc:2
  "source": "./src/js",
          ^
SyntaxError: Unexpected token :
    at exports.runInThisContext (vm.js:73:16)
    at Module._compile (module.js:443:25)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
```

Current config:
```
{
  "source": "./src/js",
  "excludes": ["jspm.config.js"],
  "destination": "./docs",
  "title": "...",
  "unexportIdentifier": true,
  "coverage": false,
  "plugins": [
    {"name": "esdoc-es7-plugin"},
    {"name": "esdoc-importpath-plugin",
      "option": {
        "replaces": [
          {"from": "^src/", "to": "/"}
        ]
      }
    }
  ]
}
```

**Comment by h13i32maru** [comment]
@mmoss Hi! Thanks for this issue!
Is your esdoc config file extention non ``.json``, by any chance?
If it is, please rename the file name to ``.json`` as workaround.

But I try to fix the crash.

**Comment by h13i32maru** [comment]
I published v0.2.6 that fixed this issue. Please try to use it.
